code/game/turfs/turf.dm

```diff
@@ -320,3 +332,21 @@
 	SSair.remove_from_active(T0)
 	T0.CalculateAdjacentTurfs()
 	SSair.add_to_active(T0,1)
+
+
+/turf/proc/update_cliff()
+	if(cliff_images) //Not cut_overlays() because turf smoothing gets royally fugged.
+		var/list/L = list()
+		for(var/k in cliff_images)
```

Comment:
this loop saddens me, but I've made a feature request for the language to allow us to get lists of values/keys from Associative lists, if that's added than this whole block will just be ```overlays -= cliff_images.values```